src/components/ImageUploader/src/ImageUploader.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+	<div :class="$s.ImageUploaderContainer">
+		<image-picker
+			v-if="canUploadImage"
+			:class="$s.ImageUploaderItem"
+			:multiple="canUploadMultiple"
+			:accept="accept"
+			@selectImages="selectImages"
+		/>
+		<image-selection
+			v-for="image of model"
+			:key="image.id"
+			:image="image"
+			:class="$s.ImageUploaderItem"
+			@removeImage="removeImage"
+		/>
+	</div>
+</template>
+
+<script>
+import ImagePicker from './ImagePicker.vue';
+import ImageSelection from './ImageSelection.vue';
+import { IMAGE_SELECTOR_STATUSES } from './constants';
+
+const MAX_PROGRESS = 100;
+const ID_INCREMENT = 1;
+const NO_MORE_IMAGES_COUNT = 0;
+
+export default {
+	name: 'MImageUploader',
+
+	components: {
+		ImagePicker,
+		ImageSelection,
+	},
+
+	props: {
+		model: {
```

Comment:
Why would we call this `model` and not `images`? Concerned that this could be conflated with `v-model` which has its own mechanisms.

Reply:
Yup good call.